modules/workbench_moderation.entity_xliff.inc

```diff
@@ -18,17 +18,24 @@ if (!function_exists('workbench_moderation_entity_xliff_translatable_source_alte
       // And this is a source node (i.e. it is not an unsaved target with no ID)
       if (workbench_moderation_node_type_moderated($wrapper->getBundle())) {
         // Then wrap the current revision, not the live version.
-        $node = workbench_moderation_node_current_load($wrapper->value());
-        // If this has not been saved yet it will return false, in which case just use the wrapper.
-        if ($node) {
-          $wrapper->set($node);
+        $current_node = workbench_moderation_node_current_load($wrapper->value());
+        $published_node = node_load($id);
+        if ($current_node->vid != $published_node->vid) {
+          // There is a published node and a current Draft.
+          // First force the published node to stay published
```

Comment:
- [ ] Is the node loaded by `node_load()` necessarily published?  Is it possible that the node had never been published and this would be publishing it?